src/config.ts

```diff
@@ -62,7 +63,8 @@ export class Config {
     this.signer = signer;
     this.gasLimits = gasLimits;
     this.abi = config.abi;
-    this.idx = { api: idx, ceramic };
+    this.idx = { api: idx };
+    this.ceramic = { api: ceramic };
```

Comment:
The use of "api" here is confusing as in the other cases it's used to store the api url string. I'd suggest we use `client` if we're storing a client, like for `safe.client` on line 26.